samples/highcharts/accessibility/point-description-enabled-threshold/demo.html

```diff
@@ -0,0 +1,11 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<figure class="highcharts-figure">
+    <div id="container"></div>
+    <p class="highcharts-description">The following chart demonstrates some Accessibility features of Highcharts, including use of the <code>pointDescriptionEnabledThreshold</code> option.
+    The <code>pointDescriptionEnabledThreshold</code> is set to 1 in this demo, which means the individual points are are not exposed to a screen reader if a series contains more than one point in this case.
+    This might be useful for larger series and is set by default to 200 for a chart.</p>
```

Comment:
> "This might be useful for larger series and is set by default to 200 for a chart." 

The wording suggests the limit is for a chart, so the total number of points from all chart series. How about "This is useful for large series and is set by default to 200." ?